src/java/org/apache/cassandra/db/filter/RowFilter.java

```diff
@@ -785,6 +808,16 @@ public boolean equals(Object o)
                 && Objects.equal(this.value, that.value);
         }
 
+        public String toString()
+        {
+            return "";
+        }
+
+        public String toStringRedacted()
+        {
+            return "<redacted>";
+        }
+
```

Comment:
Question: Why are we redacting all parameters? Parameter values contain important information about the query.

Reply:
maybe we can keep the numbers, I think that the only "privacy sensitive" data are strings or collections/structures that may contain strings.
we can keep the numbers and vectors for instance, but if we don't know the datatype then it is fine to redact everything

Reply:
what happend with query with binding variables ? are they represented here with the actual value ?

Reply:
`what happend with query with binding variables ? are they represented here with the actual value ?`
Yes, they are; the "test" that I added was intentional with a prepared statement.

```
maybe we can keep the numbers, I think that the only "privacy sensitive" data are strings or collections/structures that may contain strings.
we can keep the numbers and vectors for instance, but if we don't know the datatype then it is fine to redact everything
```
That sounds like a good idea. I will run it through Pushkala just in case